jdaviz/utils.py

```diff
@@ -175,6 +175,33 @@ def bqplot_clear_figure(fig):
     setattr(fig, 'axis_registry', {})
 
 
+def alpha_index(index):
+    """Converts an index to A-Z, AA-ZZ
+
+    Parameters
+    ----------
+    index : Integer
+        index between 9 and 701
+
+    Returns
+    -------
+    String
+        String in the range A-Z, AA-ZZ
+    """
+    # if we ever want to support more than 702 layers, then we'll need a third
+    # "digit" and will need to account for the horizontal space in the legends
+    if not isinstance(index, int):  # pragma: no cover
```

Comment:
Would still be nice to test the exceptions but as a separate test function from the parametrized one. Sorry for the extra work!

Reply:
no, I decided to remove it.  I don't think layer-lettering is a good place to enforce a maximum number of layers with a confusing traceback.  As noted in the comments and the description of the PR, this will start to introduce special characters after index 701 (which was the previous behavior after 25).  If that becomes a common use-case, then we can extend this logic to three (or N) digits and will need to update the styling of all components to handle the extra horizontal space required.